winlogbeat/_meta/fields.common.yml

```diff
@@ -31,6 +31,23 @@
         purposes. The data in the fields reported by Winlogbeat can be compared
         to the data in the XML to diagnose problems.
 
+    # For the sysmon module.
+    - name: dns
+      type: group
+      description: >
+        DNS query data.
+      fields:
+        - name: question.name
+          type: keyword
+          description: DNS query
+        - name: sysmon.status
```

Comment:
I would recommend not nesting 3rd party-named custom fields under what's about to become an ECS field set. I'd recommend nesting those under a custom top level `sysmon.*` field set.

Reply:
I left `dns.question.name` as is. And I created `sysmon.dns.answers` and `sysmon.dns.status`. I expect that the answers can be changed as soon as we finalize DNS in ECS.

Reply:
Updated this based on ECS DNS proposal.